packages/components/pie-button/src/defs.ts

```diff
@@ -1,4 +1,4 @@
-export const sizes = ['xsmall', 'small-expressive', 'small-productive', 'medium', 'large'] as const;
+export const sizes = ['xsmall', 'small-productive', 'small-expressive', 'medium', 'large'] as const;
```

Comment:
This change is just to switch the ordering of the variants in storybook, as `-productive` is a slightly smaller size than `-expressive`